source/manuals/programming-languages/css.html.md.erb

```diff
@@ -23,8 +23,7 @@ You should use the [GOV.UK Design System spacing scale](https://design-system.se
 
 ### Use with deprecated libraries
 
-If your project is not using the GOV.UK Design System, you should use the
-[variables from the frontend_toolkit][measurements.scss] or use pixel values in multiples of `5px`.
+If your project is not using the GOV.UK Design System, you should use the [variables from the frontend_toolkit][measurements.scss] or use pixel values in multiples of `5px`.
```

Comment:
Should we be talking about using toolkit at all? At the very least we should be clearly recommending using govuk-frontend instead.

Reply:
Or it could be
```suggestion
If your project is not using the GOV.UK Design System you should use pixel values in multiples of `5px`.
```

Reply:
Yeah, that seems good 👍

Reply:
Done!